assets/js/tour.js

```diff
@@ -29,12 +29,14 @@ jQuery( document ).ready(
 		jQuery( document ).on( 'click', '.pulse', function() {
 			var wrapper = jQuery( this ).closest( '.pulse-wrapper' );
 			var tourName = wrapper.data( 'tourname' );
-			var nextItem = 1 + wrapper.data( 'tourindex' );
+			var currentTourIndex = wrapper.data( 'tourindex' );
+			var nextItem = 1 + currentTourIndex;
 			var tourEndsHere = typeof window.tour[ tourName ][ nextItem ] === 'undefined';
-			var showPreviousBtn = wrapper.data( 'tourindex' ) > 1;
+			var showPreviousBtn = currentTourIndex > 1;
 			var popoverContent = wrapper.data( 'popover-content' );
 			var item;
-
+			var istopOffsetDefined = typeof window.tour[ tourName ][ currentTourIndex ].topOffset !== 'undefined' && Number.isInteger( window.tour[ tourName ][ currentTourIndex ].topOffset );
```

Comment:
Hm, I thought we were talking about `placement`? Possible values: `left` `right` `bottom` `top` `auto`, even a function would work but I am not sure this can be in a JSON.

Reply:
Oh, there are more like `left-top` or `top-left` and other combinations. See https://github.com/GlotPress/GlotPress/blob/develop/assets/js/jquery.webui-popover.js#L838

Reply:
Yes, I tried ```placement``` and the popover still covered some tour elements and that's the reason I opted for ```offsetTop```.

Reply:
On another thought, perhaps we can have both. I will add another commit that allows for setting the ```placement``` too.